docs/detections/alerts-visualizations.asciidoc

```diff
@@ -4,32 +4,45 @@
 Visualize and group detection alerts by specific parameters in the visualization section of the Alerts page. 
 
 [role="screenshot"]
-image::images/alert-page-visualizations.png[]
+image::images/alert-page-visualizations.png[Alerts page with visualizations section highlighted]
 
-Use the left menu to select a view type (*Trend*, *Table*, or *Treemap*), and use the right menus to select the ECS fields to use for grouping: 
+Use the left buttons to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the right menus to select the ECS fields to use for grouping: 
 
-* *Group by*: Primary field for grouping alerts.
+* *Top alerts by* or *Group by*: Primary field for grouping alerts.
 * *Group by top* (if available): Secondary field for further subdividing grouped alerts.
 
 For example, you can group first by rule name (`Group by: kibana.alert.rule.name`), then by host name (`Group by top: host.name`) to visualize which detection rules generated alerts, and which hosts triggered each of those rules. For groupings with a lot of unique values, the top 1,000 results are displayed.
 
-NOTE: The *Group by top* option isn't available in all view types. You can also leave *Group by top* blank to group by only the primary field in *Group by*.
+NOTE: Some view types don't have the *Group by top* option. You can also leave *Group by top* blank to group by only the primary field in *Group by*.
 
-To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when hovering over the area).
+To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when you hover over the area).
+
+=== Summary
+
+The summary visualization is the default visualization on the Alerts page and shows how alerts are distributed across a few different indicators:
```

Comment:
This is a great explanation of the visualizations.

Reply:
Yeah this is 🚀 🎸 😎